contracts/staking/StakingMinter.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.6.12;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+interface IERC20Mintable is IERC20 {
+  function mint(address to, uint256 amount) external;
+}
+
+contract StakingMinter is Ownable {
+    using SafeMath for uint256;
+    IERC20Mintable public synapse;
+    IERC20 public sSYN;
+    uint256 public synapsePerSecond;
+
+    constructor(IERC20Mintable _synapse, IERC20 _sSYN) public {
+        synapse = _synapse;
+        sSYN = _sSYN;
+    }
+
+    function setSynapsePerSecond(uint256 _rate) external onlyOwner {
```

Comment:
Consider adding:

```solidity
  sSYN.distribute()
```

Just in case the `synapsePerSecond` needs to be adjusted while the staking is active.